packages/extension/src/components/pages/ShareNFT/ShareNFT.tsx

```diff
@@ -85,20 +95,20 @@ export const ShareNFT: FC = () => {
   );
 
   const handleReject = useCallback(() => {
-    handleSendMessage(null);
+    handleSendMessage({});
```

Comment:
Is this working?
Because as per the code it seems that handleSendMessage will be returning everytime: `type: ResponseType.Response,`